stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardscan/exception/InvalidStripePublishableKeyException.kt

```diff
@@ -0,0 +1,5 @@
+package com.stripe.android.stripecardscan.cardscan.exception
+
+import java.lang.Exception
+
+class InvalidStripePublishableKeyException(message: String) : Exception(message)
```

Comment:
is this used anywhere?

Reply:
Yep, in the CardScanActivity when we check for valid params.